libbeat/publisher/queue/diskqueue/consumer.go

```diff
@@ -86,10 +86,14 @@ func (batch *diskQueueBatch) Count() int {
 	return len(batch.frames)
 }
 
-func (batch *diskQueueBatch) Event(i int) interface{} {
+func (batch *diskQueueBatch) Entry(i int) interface{} {
 	return batch.frames[i].event
 }
 
+func (batch *diskQueueBatch) ID(i int) queue.EntryID {
+	return 0
```

Comment:
Am I correct in assuming a "real" event ids for the disk queue would be in a follow on PR?  If so can we open an issue and make sure it is marked as "related" in this PR?

Reply:
Yes, the issue is https://github.com/elastic/elastic-agent-shipper/issues/27